ui/src/app/views/workspace/component.html

```diff
@@ -5,26 +5,34 @@
 </div>
 <div class="workspace-container">
   <div class="main-body">
-    <h2 class="greeting">{{showTip ? 'Get Started' : 'Recent Work'}}</h2>
-    <div style="width: 200px">
-      <div role="tooltip" aria-haspopup="true" class="tooltip tooltip-md tooltip-top-right">
-        <span *ngIf="!writePermission" class="tooltip-content">Write permission required to create cohorts</span>
-        <div class="card add-card" [ngClass]="{'disabled' : !writePermission}" (click)="!writePermission || buildCohort()">
-          <div class="create-text">
-            Create a <br/>New Cohort
-            <clr-icon shape="plus-circle" class="plus-circle"></clr-icon>
+    <div style="display: flex; flex-direction: row">
+      <h2 class="greeting" style="width: 200px">Actions</h2>
+      <h2 class="greeting" style="padding-left: 10%; margin-left:2rem">Recently Modified Items</h2>
```

Comment:
Per recent Invision comments, drop the "Items", it's cleaner: https://projects.invisionapp.com/d/#/console/12820961/303198606/comments